blueprints/in-repo-addon/index.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
 
   afterInstall: function(options) {
     var packagePath = path.join(this.project.root, 'package.json');
-    var contents    = JSON.parse(fs.readFileSync(packagePath, { encoding: 'utf8' }));
+    var contents    = fs.readJsonSync(packagePath, { encoding: 'utf8' });
```

Comment:
I'm not sure. `fs-extra` itself is not adding an encoding by itself, so I just kept the existing encoding options for now.